Cargo.toml

```diff
@@ -2,7 +2,8 @@
 name = "ulid"
 version = "1.0.0"
 authors = ["dylanhart <dylan96hart@gmail.com>"]
-edition = "2018"
+edition = "2021"
```

Comment:
I think this is for the question mark syntax in `serde?/std` and `uuid?/std`. I suspect it is needed for that, otherwise I might have changed it because there shouldn't be anything that supports the optional-dependency-feature syntax but doesn't support 2021.